### PR TITLE
fix(content): Various Gegno Corroboration Fixes + Other Tweaks

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

This PR addresses the a few unintended behaviors that came up on discord via gameplay with some minor tweaks on the side.

## Summary
With the merge of #8750, some players have found some oversights and bugs already. This PR addresses them.

Conversations now offer at proper times.
Missions no longer offer when the relevant ones have passed.
Etc..

Also threw in a few fleet tweaks.

### CODE REVIEW

File: data/gegno/gegno I corroboration.txt [hunk 1]
```diff
@@ -327,6 +337,7 @@ mission "Gegno Suspicions"
 	to offer
 		"gegno: encounters" >= 3
 		"gegno: encounters" < 6
+		not "Gegno Apprehension: offered"
```
Comment (Saugia) [inline data/gegno/gegno I corroboration.txt:340]:
```suggestion
		not "Gegno Intervention: offered"
```

File: data/gegno/gegno I corroboration.txt [hunk 2]
```diff
@@ -545,7 +557,7 @@ mission "Return to Giaru Gegno"
 
 mission "Gegno Anticipation"
 	landing
-	description "Visit <destination>."
+	description "Present yourself to the Gegno on <destination> within the next 8 days, where you will find out if you have any future with them."
```
Comment (TheGiraffe3) [inline data/gegno/gegno I corroboration.txt:560]: Also notice this is missing a name.
```suggestion
	name "Summons from the Gegno"
	description "Present yourself to the Gegno on <destination> within the next 8 days, where you will find out if you have any future with them."
```

File: data/gegno/gegno I corroboration.txt [hunk 3]
```diff
@@ -893,6 +905,7 @@ mission "Gegno Apprehension"
 mission "Gegno Intervention"
 	landing
 	clearance
+	description "Present yourself to the Gegno on <destination> within the next 8 days, where you will find out if you have any future with them."
```
Comment (TheGiraffe3) [inline data/gegno/gegno I corroboration.txt:908]:
```suggestion
	name "Summons from the Gegno"
	description "Present yourself to the Gegno on <destination> within the next 8 days, where you will find out if you have any future with them."
```

File: data/gegno/gegno fleets.txt [hunk 4]
```diff
@@ -433,6 +433,3 @@ fleet "Scin Science"
 	variant
 		"Kyanite (Guile)"
 		"Tridymite"
```
Comment (TheGiraffe3) [inline data/gegno/gegno fleets.txt:435]:
```suggestion
		"Tridymite"

```

**Review by TheGiraffe3 — CHANGES_REQUESTED** [review]

**Comment by TheGiraffe3** [comment]
The problem with this as opposed to #10302 is that this PR will take a long time to finish and merge.

**Comment by Saugia** [comment]
Gonna leave this as a draft for another day or so in case anything else pops up.

File: data/gegno/gegno I corroboration.txt [hunk 5]
```diff
@@ -533,10 +544,11 @@ mission "Return to Giaru Gegno"
 		or
 			has "event: gegno: quarg talks"
 			"gegno: encounters" >= 6
+		not "Gegno Apprehension: offered"
```
Comment (Saugia) [inline data/gegno/gegno I corroboration.txt:547]:
```suggestion
		not "Gegno Intervention: offered"
```

**Review by TheGiraffe3 — APPROVED** [review]

**Pull request merged**